shared/src/common/enums/register/register-avatar-type.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum RegisterAvatarType {
+  PNG = 'image/png',
+  JPG = 'image/jpg'
+}
```

Comment:
It would be better to create the enum for this 
//shared/common/enums/file/content-type.enum.ts

```
enum ContentType {
  PDF = 'application/pdf',
  OCTET_STREAM = 'application/octet-stream',
  JSON = 'application/json',
  JPEG = 'image/jpeg',
  PNG = 'image/png'
}
```